app/assets/scripts/views/donate.js

```diff
@@ -0,0 +1,260 @@
+                    ecosystem.
+                  </p>
+                  <p className="fold__action">
+                    <a
+                      href="https://openaq.medium.com/now-accepting-applications-for-the-openaq-community-ambassador-program-c0aee81735d5"
+                      title="Learn more"
+                      className="go-link"
+                    >
+                      <span>Learn more about this program</span>
+                    </a>
+                  </p>
+                </div>
+              </header>
+              <figure className="fold__media">
+                <aside className="aside-highlight">
+                  <div className="aside-highlight__contents">
+                    <div className="aside-highlight__prose">
+                      <p>
+                        <span>“</span>It&apos;s been a great three months since
+                        I joined the OpenAQ Ambassador Program I have been
+                        connected to and have been learning from leaders from
+                        across the world that are fighting air inequality from
+                        within their communities. OpenAQ through it&apos;s
+                        centralized data platform provides access to open air
+                        quality data from around the globe. It&apos;s so much
+                        easier now to compare air quality between cities and
+                        even countries.<span>”</span>
+                      </p>
+                      <p className="donate-quote-credit">- Farah Kazi</p>
+                    </div>
+                  </div>
+                </aside>
+              </figure>
+            </div>
+          </section>
+          <section className="fold fold--semi-light fold--type-a">
+            <div className="inner">
+              <header className="fold__header">
+                <h2>
+                  Wear a cool OpenAQ t-shirt that helps us fight air inequality!
+                </h2>
+                <div className="fold__teaser prose prose--responsive">
+                  <p>
+                    This June 2021, in honor of World Environment Day, we would
+                    like to celebrate the OpenAQ Community and support efforts
+                    to fight air inequality by releasing some new OpenAQ swag
+                    available for a limited time only! Your purchase will help
+                    support our work in democratizing access to air quality data
+                    to enable community action to fight air pollution. Please
+                    join us in wearing OpenAQ shirts with pride, and spreading
+                    the word of Clean Air for All!
+                  </p>
+                  <p className="fold__action">
+                    <a
+                      href="https://www.customink.com/fundraising/2021-openaq-summer-swag-fundraiser?side=front&type=1&zoom=false"
```

Comment:
@sruti final t-shirt link goes here when it is ready

Reply:
This is final!